File: src/main/java/core/gui/comp/table/BooleanColumn.java
package core.gui.comp.table;

import core.model.player.Player;

public class BooleanColumn extends UserColumn {

	public BooleanColumn(int id, String name, String tooltip, int minWidth){
		super(id,name,tooltip);
		this.minWidth = minWidth;
		preferredWidth = minWidth;
	}

	public Boolean getValue(Player player){

		return player.getCanBeSelectedByAssistant();
	}


}

File: src/main/java/core/gui/comp/table/BooleanColumn.kt
package core.gui.comp.table

import core.model.player.Player

class BooleanColumn(id: Int, name: String?, tooltip: String?, minWidth: Int) : UserColumn(id, name, tooltip) {
    init {
        this.minWidth = minWidth
        preferredWidth = minWidth
    }

    fun getValue(player: Player): Boolean {
        return player.canBeSelectedByAssistant
    }
}
